src/query/management/src/role/role_api.rs

```diff
@@ -44,7 +44,7 @@ pub trait RoleApi: Sync + Send {
     /// If a role is dropped, but the owner object is exists,
     ///
     /// The owner role need to update to account_admin.
-    async fn transfer_ownership_to_admin(&self, object: &OwnershipObject) -> Result<()>;
+    async fn transfer_ownership_to_admin(&self, object: &Vec<OwnershipObject>) -> Result<()>;
```

Comment:
`objects: &[OwnershipObject]` would be better

Reply:
Yes new set commit fix it